packages/react-effect/src/server.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {renderToStaticMarkup} from 'react-dom/server';
```

Comment:
don't we want to use `renderToString` so that we can hydrate the result?

Reply:
This is just for extraction, right? So we don't actually render the result. That said, using `renderToString` is a neat idea, since it does eliminate a single tree traversal. But is it as performant per traversal?

Reply:
Yeah, I think this is a reasonable API since it matches Apollo, and gives the flexibility to the user to choose how to handle it.